### PR TITLE
RWAHS-83 - accession number pattern configuration to be tied to object record type

### PR DESCRIPTION
**Pull request by kehh**

### CODE REVIEW

**Comment by kehh** [comment]
http://192.168.2.163/rwahs/providence/index.php/find/SearchObjects/Index/search/%252A has some example records matching the example patterns for each type

File: conf/multipart_id_numbering.conf [hunk 1]
```diff
@@ -29,6 +28,216 @@ formats = {
...
+				}
+			}
+		},
+		publicMemorial = {
+			# eg MA1937.90.37a, MA2015.7a-l, MA2015.40ab, MA2013.34a, MA2015.2
+			separator =,
+			elements = {
+				accession_prefix = {
+					type = CONSTANT,
+					value = MA,
```
Comment (leftclickben) [inline conf/multipart_id_numbering.conf]: Copy pasta?  Or do memorials and artefacts have the same prefix?  (Also on line 201).

**Comment by leftclickben** [comment]
Just one question.

File: conf/multipart_id_numbering.conf (hunk 1 above)
Comment (kehh) [inline conf/multipart_id_numbering.conf]: copia pasta

Comment (leftclickben) [inline conf/multipart_id_numbering.conf]: ...and my comment ended up on the wrong line but you made the correct change :-)

**Pull request merged**